### PR TITLE
escape special characters in strings

### PR DESCRIPTION
**Pull request by bheni**

### CODE REVIEW

File: sql/types/json_encode.go [hunk 1]
```diff
@@ -8,6 +8,27 @@ import (
 	"strconv"
 )
 
+var isEscaped = [256]bool{}
```
Comment (zachmu) [inline sql/types/json_encode.go:11]: Could combine these two and use `esc, ok := escapeSeq[r]` checks, any reason not to?

**Review by zachmu — APPROVED** [review]
LGTM

File: sql/types/json_encode.go (hunk 1 above)
Comment (bheni) [reply sql/types/json_encode.go:11]: I thought it would be slower.  I could be wrong.

Comment (bheni) [reply sql/types/json_encode.go:11]: That requires a map instead of an array.

Comment (bheni) [reply sql/types/json_encode.go:11]: That requires a map instead of an array.

**Pull request merged**